src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/GeneralAccountWallet/PaymentPopup/Summary/index.js

```diff
@@ -34,4 +38,8 @@ const Summary = props => {
   );
 };
 
-export default withRouter(withStyles(useStyles)(Summary));
+const mapStateToProps = state => ({
+  services: state.serviceReducer.services,
```

Comment:
`services` will contain all the services. Instead use `serviceDetails` to get the organization_name of the current service. Refer to `src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/GeneralAccountWallet/PaymentPopup/Details/index.js`